README.md

```diff
@@ -1,81 +1,35 @@
+Deicode is a tool box for running Robust Aitchison PCA on sparse compositional omics datasets, linking specific features to beta-diversity ordination. 
 
 ## Installation
 
 To install the most up to date version of deicode, run the following command
 
-    # pip
+    # pip (only supported for Qiime >= 2018.8)
     pip install deicode
 
-    # conda
+    # conda (only supported for Qiime >= 2019.1)
     conda install -c conda-forge deicode 
 
+**Note**: that deicode is not compatible with python 2, and is compatible with Python 3.4 or later. deicode is currently in alpha. We are actively developing it, and backward-incompatible interface changes may arise.
 
-## Tutorials
-
-* [What is Robust Aitchison RPCA](https://github.com/biocore/DEICODE/blob/master/ipynb/introduction.ipynb)
-
-### Qiime2 tutorial
-
-* [Sleep Apnea Biplots](https://github.com/biocore/DEICODE/blob/master/ipynb/sleep_apnea/SleepApnea-qiime2-tutorial.ipynb)
+## Qiime2 tutorial
 
-First make sure that qiime2 is installed before installing deicode. Then run
+**Note**: the official plugin docs and tutorial can be found [here](https://library.qiime2.org/plugins/q2-deicode).
 
-```
-qiime dev refresh-cache
-```
+* [Sleep Apnea Biplots](https://github.com/biocore/DEICODE/blob/master/ipynb/sleep_apnea/SleepApnea-qiime2-tutorial.md)
 
-Once qiime2 is properly interfaced with deicode, you can import your biom tables
-into Artifacts.  Here we will be using the [Sleep Apnea dataset](https://qiita.ucsd.edu/study/description/10422)
-as an example.
-
-```
-qiime tools import \
-    --input-path qiita_10422_table.biom \
-    --output-path qiita_10422_table.biom.qza \
-    --type FeatureTable[Frequency]
-```
-You can then run the qiime2 deicode rpca-biplot commmand as follows.
-
-```
-qiime deicode rpca-biplot \
-    --i-table qiita_10422_table.biom.qza \
-    --p-min-feature-count 10 \
-    --p-min-sample-count 500 \
-    --o-biplot ordination.qza
-```
-Once you have this, you can directly visualize this in emperor
-```
-qiime emperor biplot \
-    --i-biplot ordination.qza \
-    --m-sample-metadata-file qiita_10422_metadata.tsv \
-    --m-feature-metadata-file taxonomy.qza \
-    --o-visualization biplot.qzv \
-    --p-number-of-features 8
-```
-You can view the resulting visualization at https://view.qiime2.org.
-It should look as follows
-![biplot](https://github.com/biocore/DEICODE/blob/master/ipynb/sleep_apnea/qiime_view.png)
+## Other Resources
 
-### Python Tutorial
+* [What is Robust Aitchison RPCA](https://github.com/biocore/DEICODE/blob/master/ipynb/introduction.ipynb)
 
-* [Sleep Apnea Log Ratio Tutorial](https://github.com/biocore/DEICODE/blob/master/ipynb/sleep_apnea/SleepApnea-python-tutorial.ipynb)
+- The code for OptSpace was translated to python from a [MATLAB package](http://swoh.web.engr.illinois.edu/software/optspace/code.html) maintained by Sewoong Oh (UIUC).
+- Transforms and PCoA : [Scikit-bio](https://github.com/biocore/scikit-bio)
```

Comment:
I would link to http://scikit-bio.org instead.

Reply:
changed, nice catch. thanks!